packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/apexExecutionOverlayActionCommand.test.ts

```diff
@@ -5,15 +5,13 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { FIELD_INTEGRITY_EXCEPTION } from '@salesforce/salesforcedx-apex-replay-debugger/out/src/index';
```

Comment:
I noticed that in several of the PRs, that the "out" import paths have been fixed.  Can this import path be changed to just @salesforce/salesforcedx-apex-replay-debugger?

Reply:
I'll create a ticket to address it for each individual package to avoid bloating this PR anymore then it already is :).  It's a simple change as it's just adding a 'main' property to the package.json for the '@salesforce/salesforcedx-apex-replay-debugger' module, but it'll touch every file where we import it.